layers/+lang/python/README.org

```diff
@@ -179,3 +180,15 @@ If you want to customize the fill column value, use something like this inside t
 #+BEGIN_SRC elisp
 (setq python-fill-column 99)
 #+END_SRC
+** Sort imports
+If you want imports to be automatically sorted when you save a file, use something like this inside the ~user-init~ or ~user-config~ function in your ~.spacemacs~:
+
+#+BEGIN_SRC elisp
+(setq python-sort-imports t)
```

Comment:
Would be better to mention that you can add this in layer's config section where you configure variables as per spacemacs' convention. 
```elisp
  (setq-default dotspacemacs-configuration-layers '(
    (python :variables python-sort-imports t)))
```